snippets/csharp/System.Linq/Queryable/AggregateTSource/queryable.cs

```diff
@@ -1318,14 +1333,14 @@ static void LastOrDefaultEx1()
         {
             // <Snippet45>
             // Create an empty array.
-            string[] fruits = { };
+            string[] fruits = [];
 
             // Get the last item in the array, or else the default
             // value for type string (null).
-            string last = fruits.AsQueryable().LastOrDefault();
+            var last = fruits.LastOrDefault();
 
             Console.WriteLine(
-                String.IsNullOrEmpty(last) ? "[STRING IS NULL OR EMPTY]" : last);
+                last ?? "[STRING IS NULL OR EMPTY]");
```

Comment:
Same comment I had on the enumerable sample. This does change the semantics of the sample.